### PR TITLE
DOCS-12649 rtfctl-docs-update

### PR DESCRIPTION
**Pull request by IsaacEldridge**
1. Restructured the page and separated commands into categories: installing, upgrading, configuring, managing apps, and troubleshooting
2. Added missing commands based on rtfctl repo

@kmendiratta-crm @jeff-1amstudios @sarjeet2013

### CODE REVIEW

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 1]
```diff
@@ -52,70 +52,534 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+----
+
+=== `restore`
+Restores local Runtime Fabric state from a backup 
+
+----
+rtfctl restore /var/backups/<filename>.tar.gz
+----
+
+=== `upgrade-cluster`
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/install-rtfctl.adoc]: We can remove this section and any references to it. Its an hidden, deprecated command

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 2]
```diff
@@ -52,70 +52,534 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+|Command |Description
+|xref:delete[`delete`] | Deletes a secure property
+|xref:describe[`describe`] | Displays details for an application
+|xref:disk[`disk`] | Displays disk usage for an application
+|xref:memory[`memory`] | Displays the JVM memory usage in a pod
+|xref:package[`package`] | Creates a debugging information package for an application
+|xref:restart[`restart`] | Restarts an application
+|===
+
+=== `delete`
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/install-rtfctl.adoc:322]: This should probably be in the same group as `apply secure-property`

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 3]
```diff
@@ -52,70 +52,534 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+|xref:heapdump[`heapdump`] | Triggers a JVM heap dump
+|xref:threaddump[`threaddump`] | Triggers a JVM thread dump
+|xref:get[`get`] | Displays one or more resources
+|xref:ping[`ping`] | Tests network connectivity
+|xref:report[`report`] | Generates a diagnostics report
+|xref:status[`status`] | Displays the status of Runtime Fabric
+|xref:test[`test`] | Verifies outbound network connectivity to the Runtime Fabric control plane
+|===
+
+=== `heapdump`
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/install-rtfctl.adoc:394]: I would put heapdump and threaddump in the application section - they are troubleshooting commands, but specific to applications, not the Runtime Fabric cluster itself

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 4]
```diff
@@ -52,70 +52,534 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
 [%header%autowidth.spread,cols="a,a"]
-.rtfctl Flags
 |===
-|Flag |Description
-| -h, --help | Displays `rtfctl` help information.
-| -n, --namespace string | If present, displays the namespace scope.
-| --pod string | If present, displays the pod name.
+|Command |Description
+|xref:install[`install`] | Installs Runtime Fabric
+|xref:reinstall[`reinstall`] | Re-installs Runtime Fabric components
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/install-rtfctl.adoc:248]: lets remove this section - that command should actually be a subcommand under `appliance`

File: modules/ROOT/pages/install-rtfctl.adoc (hunk 2 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/install-rtfctl.adoc:322]: I think in the case of this structure it doesn't make sense to put them together. And we do have a separate page for secure properties.

File: modules/ROOT/pages/install-rtfctl.adoc (hunk 3 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/install-rtfctl.adoc:394]: ok, moved

**Review by jeff-1amstudios — DISMISSED** [review]

**Review by jeff-1amstudios — DISMISSED** [review]

**Review by kmendiratta-crm — APPROVED** [review]

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 5]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+== Commands for Managing the Runtime Fabric Appliance
+
+Use the following commands to manage the Runtime Fabric Appliance.
+
+[NOTE]
+These commands are for Runtime Fabric on VMs / Bare Metal only.
+
+=== `appliance` 
+
+Performs operations to the Runtime Fabric appliance
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:68]: on the? 
that target the?

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 6]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+|xref:apply[`apply`] | Applies changes to a Runtime Fabric configuration
+|xref:update[`update`] | Updates rtfctl
+|xref:wait[`wait`] | Waits for Runtime Fabric components to become ready
+|===
+
+=== `apply` 
+
+Applies changes to a Runtime Fabric configuration.
+
+* Applies a HTTP proxy configuration
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:140]: an HTTP (throughout)

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 7]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+
+=== `wait`
+Waits for Runtime Fabric components to become ready
+
+* Specifies the number of seconds to wait 
+----
+rtfctl wait <value in seconds>
+----
+
+* Specifies the Number of seconds to pass before timing out the wait (Default is 600 seconds)
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:234]: number of seconds

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 8]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
 [%header%autowidth.spread,cols="a,a"]
-.rtfctl Flags
 |===
-|Flag |Description
-| -h, --help | Displays `rtfctl` help information.
-| -n, --namespace string | If present, displays the namespace scope.
-| --pod string | If present, displays the pod name.
+|Command |Description
+|xref:install[`install`] | Installs Runtime Fabric
+|xref:reinstall[`reinstall`] | Re-installs Runtime Fabric components
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:248]: I think our style would say Reinstalls (no hyphen). (Throughout)

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 9]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+----
+
+=== `backup`
+Performs a back up of the local Runtime Fabric state.
+----
+rtfctl backup /var/backups/<backup-filename>.tar.gz
+----
+
+=== `restore`
+Restores local Runtime Fabric state from a backup
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:285]: the local

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 10]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+
+=== `restore`
+Restores local Runtime Fabric state from a backup 
+
+----
+rtfctl restore /var/backups/<filename>.tar.gz
+----
+
+=== `validate`
+Validates the cluster for Runtime Fabric installation
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:292]: the Runtime
or
a Runtime

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 11]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+
+Displays disk usage for an application
+
+* Displays the disk usage for an application, using the first replica by default
++
+----
+rtfctl disk <app-name>
+----
+
+* Displays the disk usage for an application, using a specific replica
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:345]: no comma

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 12]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+=== `memory`
+Displays the JVM memory usage in a pod
+
+* Gets the JVM memory for each replica
++
+----
+rtfctl memory <app-name>
+----
+
+* Gets the JVM memory, for a particular replica
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:360]: no comma

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 13]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
++
+----
+rtfctl memory <app-name> --pod <app-name>-pod-1
+----
+
+=== `package` 
+Creates a debugging information package for an application
+
+* Creates a package for an application, using the first pod by default
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:369]: no comma

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 14]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
++
+----
+rtfctl package <app-name>
+----
+
+* Created a package for an application containing the apps, policies, and `.mule` directories
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:375]: Creates?

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 15]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+* Performs a JVM heap dump for an application, saving it as `/tmp/dump.hprof`, using a specific replica
++
+----
+rtfctl heapdump <app-name> /tmp/dump.hprof --pod <replica-name>
+----
+
+=== `threaddump`
+Triggers a JVM threaddump
+
+* Performs a JVM thread dump for an application, using the first replica by default
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:412]: No comma

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 16]
```diff
@@ -52,70 +52,517 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
...
+. Follow the instructions to xref:install-manual#base64-encode-your-mule-license-key[Base64 Encode your Mule License Key].
+. Run the following command:
++
+----
+$ sudo ./rtfctl apply mule-license '<BASE64-encoded-license>'
 Updating rtf namespace... OK
 ----
 
-=== Take a Heap Dump
+=== Capturing a Heap Dump
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/install-rtfctl.adoc:558]: Capture a Heap Dump
Our standard says not to use gerunds in headings

**Review by bjoson-mulesoft — APPROVED** [review]

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**